src/components/account.js

```diff
@@ -4,17 +4,25 @@ import { fetchUsersServices } from '../actions';
 import ServiceCardListRow from './service_card_list_row.js';
 
 class Account extends Component {
+
+    constructor(props) {
+        super(props)
+        this.state = {
+            user :  this.props.user || JSON.parse(localStorage.getItem('loggedInUser')),
```

Comment:
why not use the getUserFromLocalStorage action

Reply:
and call it in component did mount. Then only use this.props.user

Reply:
Lemme try that rq

Reply:
Like this?
```javascript

    componentDidMount() {
        this.props.getUserFromLocalStorage();
        this.props.fetchUsersServices(this.props.user._id);
    }
```
Getting a TypeError: Cannot read property '_id' of undefined.

I thought the this.props would be updated by the reducer on the second render? The issue is happening on the initial render. Would have preferred the action creator though.

Reply:
add if statement if user is in props then create the action

Reply:
also probably need to add this before going to fetch it
`        if (localStorage.getItem('loggedInUser'))
            this.props.getUserFromLocalStorage();  `

Reply:
to see if we even have that in localstorage